src/v2/Components/FlashBanner/__tests__/FlashBanner.jest.tsx

```diff
@@ -1,32 +1,189 @@
-import { mount } from "enzyme"
 import "jest-styled-components"
 import React from "react"
 import { FlashBanner } from "v2/Components/FlashBanner"
+import { graphql } from "react-relay"
+import { Banner } from "@artsy/palette"
+import { flushPromiseQueue, renderRelayTree } from "v2/DevTools"
+import { mount } from "enzyme"
+
+jest.unmock("react-tracking")
```

Comment:
I unmocked this even though the PR does not include tracking yet. Should it?

Reply:
Best to group this stuff all together. If this PR doesn't have tracking, no need to add unaspected mock code. (And if you were gonna mock you'll want to look at how we do it in, say, the navbar component; we don't mock react-tracking, but rather our useTracking custom hook that wraps react tracking.)

Reply:
added tracking so this is out of date